src/autopas/selectors/FeatureVector.h

```diff
@@ -23,13 +23,14 @@ class FeatureVector : public Configuration {
   /**
    * Number of tune-able dimensions.
    */
-  static constexpr size_t featureSpaceDims = 4;
+  static constexpr size_t featureSpaceDims = 5;
```

Comment:
This stuff probably should be done differently.
@Njan0 Let's talk about this in the next meeting.